src/components/toast/toast.js

```diff
@@ -306,17 +330,22 @@ function MdToastProvider($$interimElementProvider) {
     })
     .addPreset('simple', {
       argOption: 'textContent',
-      methods: ['textContent', 'content', 'action', 'highlightAction', 'highlightClass', 'theme', 'parent' ],
+      methods: ['textContent', 'content', 'action', 'actionKey', 'actionHint', 'highlightAction',
+                'highlightClass', 'theme', 'parent', 'dismissHint' ],
       options: /* @ngInject */ function($mdToast, $mdTheming) {
         return {
           template:
             '<md-toast md-theme="{{ toast.theme }}" ng-class="{\'md-capsule\': toast.capsule}">' +
-            '  <div class="md-toast-content">' +
-            '    <span class="md-toast-text" role="alert" aria-relevant="all" aria-atomic="true">' +
+            '  <div class="md-toast-content" aria-live="polite" aria-relevant="all">' +
+            '    <span class="md-toast-text">' +
             '      {{ toast.content }}' +
             '    </span>' +
+            '    <span class="md-visually-hidden">{{ toast.dismissHint }}</span>' +
+            '    <span class="md-visually-hidden" ng-if="toast.action">' +
+            '      {{ toast.actionHint }}' +
+            '    </span>' +
```

Comment:
The new `dismissHint` and `actionHint` `span` elements are returned by tests that do `.find('span').text()` like `expect(parent.find('span').text().trim()).toBe('Do something');`. So the tests will now get the all of the contents (even the contents of the two new `md-visually-hidden` `span`s). This broke two AngularJS Material tests, but it appears to be breaking some tests within Google as well.

Can anyone think of any way to work around this that doesn't mean changing all of the tests?

This is in an Aria Live Region, so I don't think that something like `aria-describedby` would be a valid workaround.

I wonder if wrapping the new elements in a `div` or changing them to `div`s would work. It seems like it may break screen readers but I will test it.

- [ ] Test wrapping `dismissHint` and `actionHint` `span` elements in `divs`
- [ ] Test changing  `dismissHint` and `actionHint` `span` elements to `divs`

Reply:
Jeremy indicated that it should actually be easy to fix the 2 tests within Google that are broken by this.